packages/fast-web-utilities/src/html.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Gets the client bounding rectangle including any margins of an element.
+ */
+export function getClientRectWithMargin(element: HTMLElement): ClientRect | DOMRect {
+    if (!element) {
+        return;
+    }
+
+    const clone: any = element.getBoundingClientRect();
```

Comment:
Instead of any could this be `ClientRect | DOMRect` as you have above?

Reply:
Yep.